domino/domino.py

```diff
@@ -374,6 +379,12 @@ def throw_if_information_invalid(key: str, info: dict) -> bool:
             if "maxWorkerCount" in compute_cluster_properties and not is_comute_cluster_autoscaling_supported(self._version):
                 raise UnsupportedFieldException(f"'maxWorkerCount' is not supported in Domino {self._version}.")
 
+        def validate_is_external_volume_mounts_supported():
+            if not is_external_volume_mounts_supported:
```

Comment:
Where this variable is defined?

Reply:
it is defined in domino/helpers.py

Reply:
I just do not understand:

1. `is_external_volume_mounts_supported(version: str) -> bool` in `helpers.py` is a function.
2. What does `not is_external_volume_mounts_supported` mean?